### DISCUSSION TITLE
Error message when scanning QR code

### DISCUSSION THREAD

**Discussion by arcturiano** [BitCannaGlobal/WeedMobile#93] [General]
I am trying to send Raul some BCNA via the QR code that he shared in Discord

1. CLicked on settings button and then on Scan QR option
2. App request camera permission successfuly and I am able to scan code
3. When redirected to send token screen I see an error

Guessing this is due to me not having the 70BCNA that Raul is requesting (only have 10BCNA that I received from faucet)

This error I believe, should say "not enough BCNA" - "Not enough funds" 

![scanqrsend](https://github.com/BitCannaGlobal/WeedMobile/assets/94879750/97b5a87e-d6a5-4d55-8bcb-f5acde135823)
 or something similar

**Comment by atmoner** [comment]
@arcturiano I got it too! We are working on it

**Reply by TJ-BitCanna** [reply]
Should it be converted in an issue in that case?
